java/code/src/com/redhat/rhn/manager/system/VirtualizationActionCommand.java

```diff
@@ -37,7 +37,7 @@
 public class VirtualizationActionCommand {
 
     private static Logger log = Logger.getLogger(VirtualizationActionCommand.class);
-    private static final TaskomaticApi TASKOMATIC_API = new TaskomaticApi();
+    private static TaskomaticApi taskomaticApi = new TaskomaticApi();
```

Comment:
I know it's not part of your PR, but maybe it's worth declaring the log as `static final` as per other classes?

Reply:
done